### PR TITLE
Refactoring based on Swift lint logs

### PR DESCRIPTION
**Pull request by addb**
## Description
- Refactoring based on Swift lint logs
- Project file cleanup
- Modified swiftlint.yml for project rules

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: TestApps/TestApp/Player/VideoPlayer.swift [hunk 1]
```diff
@@ -59,7 +59,7 @@ class VideoPlayer: AVPlayer {
     let QOEINFO_STARTUPTIME: Double = 2
     let QOEINFO_FPS: Double = 24
     let QOEINFO_DROPPEDFRAMES: Double = 10
-    let VIDEO_LENGTH: Double = 1800
+    let VIDEO_LENGTH: Double = 180
```
Comment (emdobrin) [inline TestApps/TestApp/Player/VideoPlayer.swift:62]: what is the reason for changing this value?

File: .swiftlint.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@ identifier_name:
         - no
         - ok
 line_length:
-    warning: 180
-    error: 200
+    warning: 250
+    error: 300
```
Comment (emdobrin) [inline .swiftlint.yml:38]: Keeping the line length to a relatively low number can increase code readability. How many such occurrences we have, and are these occurrences in both code and tests?

File: TestApps/TestApp/Player/VideoPlayer.swift (hunk 1 above)
Comment (addb) [reply TestApps/TestApp/Player/VideoPlayer.swift]: To match the actual length of the video used in the test app

File: TestApps/TestApp/Analytics/MediaAnalyticsProvider.swift [hunk 3]
```diff
@@ -25,17 +25,16 @@ class MediaAnalyticsProvider: NSObject {
         _player = player
 
         // Pass optional configuration when creating tracker
-        /*var config: [String: Any] = [:]
-        config[MediaConstants.TrackerConfig.CHANNEL] = "custom-swift-channel" // Overrides channel configured from launch
-        _tracker = Media.createTrackerWith(config: config)*/
-
-        _tracker = Media.createTracker()
+        var config: [String: Any] = [:]
+        // config[MediaConstants.TrackerConfig.CHANNEL] = "e2e-swift-channel" // Overrides channel configured from launch
```
Comment (emdobrin) [inline TestApps/TestApp/Analytics/MediaAnalyticsProvider.swift:29]: nit: from launch -> from remote configuration

**Review by emdobrin — APPROVED** [review]

**Review by kevinlind — COMMENTED** [review]
Looks good. I wasn't able to see but have you included a build phase to the project targets to run SwiftLint on build?

**Pull request merged**